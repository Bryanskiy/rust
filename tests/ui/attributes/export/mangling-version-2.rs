//@ compile-flags: -Zunstable-options -Csymbol-mangling-version=legacy
extern dyn crate libr;
//~^ ERROR `extern dyn` annotation is only allowed with `v0` mangling scheme
//~| ERROR can't find crate for `libr`

fn main() {}
