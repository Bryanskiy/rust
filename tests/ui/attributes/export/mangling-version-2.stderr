error[E0463]: can't find crate for `libr`
  --> $DIR/mangling-version-2.rs:2:1
   |
LL | extern dyn crate libr;
   | ^^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: `extern dyn` annotation is only allowed with `v0` mangling scheme
  --> $DIR/mangling-version-2.rs:2:1
   |
LL | extern dyn crate libr;
   | ^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0463`.
